k8s/grafana/Makefile

```diff
@@ -1,19 +1,26 @@
-include ../app.Makefile
 include ../crd.Makefile
 include ../gcloud.Makefile
 include ../var.Makefile
 
-TAG ?= 5.3
+# The digest should be updated automatically, or tag <major>.<minor>.<path/build>
+# should be available on GCP Marketplace.
+# Use `.build/grafana/show-version` target to fill the RELEASE variable.
```

Comment:
This is a TODO right?
- We should name this `.build/grafana/VERSION`, and it should be an actual file (generated by this same build target) that contains the Grafana version.
- We should validate the version against the TRACK.